static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -794,17 +794,32 @@ class IssueRuleEditor extends DeprecatedAsyncView<Props, State> {
     if (!organization.features.includes('change-alerts')) {
       return this.state.configs?.conditions ?? null;
     }
+    let conditions = this.state.configs?.conditions ?? null;
 
-    return (
-      this.state.configs?.conditions?.map(condition =>
+    if (
```

Comment:
i'm wondering if i need to add this logic in here too? https://github.com/getsentry/sentry/blob/158c3c4bb13636fa8a775b2420c1fc1b412db5f5/static/app/views/alerts/rules/issue/ruleNodeList.tsx#L181 not sure where that's used

Reply:
i think change alerts should be available to most orgs? maybe?

Reply:
no sorry, i mean should my new flag be applied in this other file too. in my local testing this correctly gated the condition, but i'm not sure where this other ruleNodeList thing is used